protobuf/v1/host.proto

```diff
@@ -118,4 +118,7 @@ message StatNvmeControllerRequest {
 
 message StatNvmeControllerResponse {
   NvmeControllerIoStats stats = 1; // Controller I/O statistics
-}
\ No newline at end of file
+}
+
+// Means no arguments or no return value.
+message Null {}
```

Comment:
Why is there a need to change this? Other methods above also use the Empty struct.

Reply:
reverted it.